### DISCUSSION TITLE
Make it possible to not write REPLAYGAIN_REFERENCE_LOUDNESS tag

### DISCUSSION THREAD

**Discussion by foogolator** [complexlogic/rsgain#75] [Ideas]
Would it be possible to make it so that the REPLAYGAIN_REFERENCE_LOUDNESS tag isn't written using a presets option?
This is a workaround for gstreamer's replaygain implementation not working when this tag exists
https://gitlab.freedesktop.org/gstreamer/gstreamer/-/issues/1676 - Gstreamer issue for reference

**Comment by complexlogic** [comment]
rsgain does not write the `REPLAYGAIN_REFERENCE_LOUDNESS` tag, but it will delete that tag if it already exists in the file.
